src/core/frontend/framebuffer_layout.h

```diff
@@ -18,6 +18,12 @@ enum ScreenDocked : u32 {
     HeightDocked = 1080,
 };
 
+enum class Aspect {
+    AspectDefault,
```

Comment:
```suggestion
    Default,
```

The `Aspect` prefix is only really necessary for `21by9`, since C++ doesn't allow identifiers to begin with a number.